src/osemgrep/targeting/Git_path.mli

```diff
@@ -43,5 +45,22 @@ val make_absolute : t -> t
 val normalize : t -> (t, string) result
 val of_fpath : Fpath.t -> t
 
+(* Convert back to a system path. *)
+val to_fpath : root:Fpath.t -> t -> Fpath.t
+
 (* / *)
 val root : t
+
+(*
+   Return an absolute, normalized git path relative to the project root.
```

Comment:
We can get rid of Common2.readable, which provides a similar feature.

Reply:
It would be easier if I knew what `Common.readable` does...